dbsc/config/snakebids.yml

```diff
@@ -93,8 +94,8 @@ parse_args:
     nargs: "+"
 
   --step:
-    help: '(Mrtrix3) set the step size of the algorithm in mm 
-          (default: 0.35mm)'
+    help: '(Mrtrix3) set the step size of the algorithm in mm. Should set to
```

Comment:
If we're going to include this recommendation it would be good to briefly explain why or give a reference.